# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hermes/gcc4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur Hermes-1.3.3.orig/src/HeadMMX.h Hermes-1.3.3/src/HeadMMX.h
--- Hermes-1.3.3.orig/src/HeadMMX.h	2000-12-06 06:55:28.000000000 +0200
+++ Hermes-1.3.3/src/HeadMMX.h	2005-10-10 18:00:10.000000000 +0300
@@ -56,6 +56,8 @@
   extern "C" {   
   #endif
 
+#if 0
+
   void ConvertMMX(HermesConverterInterface *) __attribute__ ((alias ("_ConvertMMX")));
   void ClearMMX_32(HermesClearInterface *) __attribute__ ((alias ("_ClearMMX_32")));
   void ClearMMX_24(HermesClearInterface *)  __attribute__ ((alias ("_ClearMMX_24")));
@@ -70,6 +72,8 @@
   void ConvertMMXpII32_16RGB555() __attribute__ ((alias ("_ConvertMMXpII32_16RGB555")));
   void ConvertMMXpII32_16BGR555() __attribute__ ((alias ("_ConvertMMXpII32_16BGR555")));
 
+#endif
+
   #ifdef __cplusplus
   }
   #endif
diff -Nur Hermes-1.3.3.orig/src/HeadX86.h Hermes-1.3.3/src/HeadX86.h
--- Hermes-1.3.3.orig/src/HeadX86.h	2000-12-06 06:55:28.000000000 +0200
+++ Hermes-1.3.3/src/HeadX86.h	2005-10-10 17:55:06.000000000 +0300
@@ -88,6 +88,8 @@
   extern "C" {
   #endif
 
+#if 0
+
   int Hermes_X86_CPU() __attribute__ ((alias ("_Hermes_X86_CPU")));
 
   void ConvertX86(HermesConverterInterface *) __attribute__ ((alias ("_ConvertX86")));
@@ -137,6 +139,8 @@
   extern int32 ConvertX86p16_32RGBA888_LUT_X86[512] __attribute__ ((alias ("_ConvertX86p16_32RGBA888_LUT_X86")));
   extern int32 ConvertX86p16_32BGRA888_LUT_X86[512] __attribute__ ((alias ("_ConvertX86p16_32BGRA888_LUT_X86")));
 
+#endif
+
   #ifdef __cplusplus
   }
   #endif
